Assets/Scripts/Interactable.cs

```diff
@@ -58,4 +76,29 @@ void setIrritationScore(IrritationReaction irritation)
       Debug.Log($"{npc.name} didn't see you messing with {this}...");
     }
   }
+
+  protected virtual void MouthInteraction()
+  {
+    Debug.Log("MouthInteraction");
+  }
+
+  protected virtual void PawInteraction()
+  {
+    Debug.Log("PawInteraction");
+  }
+
+  protected virtual void BodyInteraction()
+  {
+    Debug.Log("BodyInteraction");
+  }
```

Comment:
This is where behavior shared across all interactions will live. It should be pretty minimal. The audio code will live here and maybe anything we need to update the game state later